python/pyiceberg/utils/datetime.py

```diff
@@ -38,6 +38,11 @@ def micros_to_days(timestamp: int) -> int:
     return timedelta(microseconds=timestamp).days
 
 
+def micros_to_date(timestamp: int) -> date:
+    """Converts a timestamp in microseconds to a date"""
+    return days_to_date(timedelta(microseconds=timestamp).days)
```

Comment:
Is it better to embed the body of `micros_to_days` or to just call that method?

Reply:
It is not used, so I'll remove it for now! 👍🏻

Reply:
Removed :)